runtime/src/main/java/io/quarkiverse/resteasy/problem/openapi/HttpProblemSchema.java

```diff
@@ -0,0 +1,45 @@
+package io.quarkiverse.resteasy.problem.openapi;
+
+import java.net.URI;
+
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+@Schema(name = "HttpProblem", description = "HTTP Problem Response according to RFC9457 & RFC7807")
+public class HttpProblemSchema {
```

Comment:
BTW. Although I'm not "happy" 😉  looking at this as kind of duplicate of `HttpProblem` I do prefer this than exposing (even correctly set-up) `HttpProblem` directly aka _never leak your implementation_.

👏

Reply:
Not perfect, not terrible as they say :) Maybe for v4.0.0 release we can unify this.

Reply:
TIL (well, today I understood, I learned few days ago) that JVM class loader does not throw `ClassNotFoundExceptions` for annotations that are not present in the classpath, and this is as-designed behaviour O_o
https://stackoverflow.com/questions/3567413/why-doesnt-a-missing-annotation-cause-a-classnotfoundexception-at-runtime

This means we can add these annotations directly on the main `HttpProblem` class without breaking backward compatibility (and without adding non-optional dependency). I'm gonna test it before this PR is ready for review, and maybe HttpProblem and HttpProblemSchema become one.

Reply:
@pazkooda I've just pushed this change, now `HttpProblem` itself is annotated with openapi stuff so duplication is kind of gone. I'm not sure if I understood your original comment about exposing `HttpProblem` directly so not sure how/if you'll like this change/proposal. In the end `HttpProblem` is already exposed 'everywhere' and is part of public programmatic api of this extension.